doc/arkode/guide/source/Usage/SplittingStep/SplittingStepCoefficients.rst

```diff
@@ -0,0 +1,432 @@
+   :return: A :c:type:`SplittingStepCoefficients` structure if successful or a
+      ``NULL`` pointer if an argument was invalid or an allocation error
+      occurred.
+   
+   .. versionadded:: x.y.z
+
+
+.. c:function:: SplittingStepCoefficients SplittingStepCoefficients_Alloc(int sequential_methods, int stages, int partitions)
+
+   Allocates an empty SplittingStepCoefficients.
+
+   :param sequential_methods: The number :math:`r` of sequential methods
+      combined to produce the overall operator splitting solution.
+   :param stages: The number :math:`s` of stages.
+   :param partitions: The number :math:`P` of partitions in the IVP.
+   :return: A :c:type:`SplittingStepCoefficients` structure if successful or a
+      ``NULL`` pointer if an argument was invalid or an allocation error
+      occurred.
+   
+   .. versionadded:: x.y.z
+
+
+.. c:function:: SplittingStepCoefficients SplittingStepCoefficients_Create(int sequential_methods, int stages, int partitions, int order, sunrealtype* alpha, sunrealtype* beta)
+
+   Allocates a SplittingStepCoefficients and fills it with the given values.
+
+   :param sequential_methods: The number :math:`r` of sequential methods
+      combined to produce the overall operator splitting solution.
+   :param stages: The number :math:`s` of stages.
+   :param partitions: The number :math:`P` of partitions in the IVP.
+   :param order: The method order of accuracy.
+   :param alpha: An array of length ``sequential_methods`` containing the
+      weight of each sequential method used to produce the overall operator
+      splitting solution.
+   :param beta: An array of length
+      ``sequential_methods * (stages+1) * partitions`` containing the time nodes
+      of the inner integrations in the order
+
+      .. math::
+         \beta_{1,0,1}, \dots, \beta_{1,0,P}, \beta_{1,1,1}, \dots, \beta_{1,1,P}, \dots, \beta_{2,0,1}, \dots, \beta_{r,s,P}.
+
+   :return: A :c:type:`SplittingStepCoefficients` structure if successful or a
+      ``NULL`` pointer if an argument was invalid or an allocation error
+      occurred.
+   
+   .. versionadded:: x.y.z
+
+
+.. c:function:: SplittingStepCoefficients SplittingStepCoefficients_Copy(SplittingStepCoefficients coefficients)
+
+   Creates copy of the given splitting coefficients.
+
+   :param coefficients: The splitting coefficients to copy.
+   :return: A :c:type:`SplittingStepCoefficients` structure if successful or a
+      ``NULL`` pointer if an allocation error occurred.
+   
+   .. versionadded:: x.y.z
+
+
+.. c:function:: void SplittingStepCoefficients_Free(SplittingStepCoefficients coefficients)
+
+   Deallocate the splitting coefficients memory.
+
+   :param coefficients: The splitting coefficients.
+   
+   .. versionadded:: x.y.z
+
+
+.. c:function:: void SplittingStepCoefficients_Write(SplittingStepCoefficients coefficients, FILE* outfile)
+
+   Write the splitting coefficients to the provided file pointer.
```

Comment:
I agree it can be moved out of the note, but I wonder if it belongs in the `:param outfile:` description rather than the overall function description.

Reply:
Agreed, I think that would be better